ts/features/itwallet/discovery/components/ItwPrivacyWebViewComponent.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import {
+  ButtonSolid,
+  ContentWrapper,
+  IOStyles,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import WebView from "react-native-webview";
+import { View } from "react-native";
+import { WebViewSource } from "react-native-webview/lib/WebViewTypes";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+import { AVOID_ZOOM_JS, closeInjectedScript } from "../../../../utils/webview";
+import I18n from "../../../../i18n";
+import ItwMarkdown from "../../common/components/ItwMarkdown";
+
+type Props = {
+  source: WebViewSource;
+  onAcceptTos: () => void;
+  onLoadEnd: () => void;
+  onError: () => void;
+};
+
+const ItwPrivacyWebViewComponent = ({
+  source,
+  onAcceptTos,
+  onLoadEnd,
+  onError
+}: Props) => {
+  const { bottom } = useSafeAreaInsets();
+  return (
+    <View style={IOStyles.flex}>
+      <WebView
+        androidCameraAccessDisabled={true}
+        androidMicrophoneAccessDisabled={true}
+        onLoadEnd={onLoadEnd}
+        onError={onError}
+        textZoom={100}
+        style={IOStyles.flex}
+        source={source}
+        injectedJavaScript={closeInjectedScript(AVOID_ZOOM_JS)}
+      />
+
+      <View style={{ marginBottom: bottom }}>
+        <ContentWrapper>
+          <VSpacer size={4} />
+          <ItwMarkdown>
+            {I18n.t("features.itWallet.ipzsPrivacy.warning")}
+          </ItwMarkdown>
+          <ButtonSolid
```

Comment:
What do you think of adding some padding or safe area, so it is aligned with the same button in the previous screen?

<img width="240" src="https://github.com/user-attachments/assets/77dde197-e707-41c2-97f5-d615b36299eb" />

Reply:
It should be fixed, thanks!